resources/js/components/Account/PasswordStrength.vue

```diff
@@ -0,0 +1,65 @@
+<script>
+
+export default {
+    props: {
+        password: {
+            type: String,
+            default: ''
+        },
+        minClasses: {
+            type: Number,
+            default: null
+        },
+        checks: {
+            type: Object,
+            default: null
+        },
+    },
+
+    data() {
+        return {
+            strengths: [],
+            errors: [],
+        }
+    },
+
+    render() {
+        return this.$scopedSlots.default(this)
+    },
+
+    mounted() {
+        this.validate()
+    },
+
+    watch: {
+        password: {
+            handler: function () {
+                this.validate()
+            }
+        }
+    },
+
+    methods: {
+        validate() {
+            this.errors = []
+            this.strengths = []
+            for (const check in this.checks) {
+                if (Object.prototype.hasOwnProperty.call(this.checks, check)) {
```

Comment:
```suggestion
                if (this.checks.hasOwnProperty(check))) {
```

or even

```suggestion
                if (check in this.checks) {
```

However, what's this `if` even for? You're looping over the object and then you check whether the item you got from the object is in the object. It'll always return true right?